core/src/main/scala/com/permutive/refreshable/Refreshable.scala

```diff
@@ -131,6 +131,11 @@ object Refreshable {
     *   a callback invoked whenever a new value is generated, the
     *   [[scala.concurrent.duration.FiniteDuration]] is the period that will be
     *   waited before the next new value
+    * @param newValueSelector
```

Comment:
I'm not sure "selector" is the right name here as you could combine the two values (per the test)

Reply:
Yeah it is a strange name. Something like `combine` makes more sense to me

Also, do you think that it should actual be `(CachedValue[A], CachedValue[A]) => F[Unit]`? If you want this functionality at all then I feel that you might care about distinguishing between those cases

Reply:
Yeah `combine` sounds better. Good shout on the more powerful function, I suppose less powerful functions can esily be derived where needed.

Reply:
Actually, thinking again about this, I'm not sure we should allow people to modify the outer `CachedValue` as it communicates state, but it's probably useful to help decide which value to take. So the function should probably look like `(CachedValue[A], CachedValue[A]) => F[A]`

Reply:
🤦 yes, I meant `F[A]` 😂